src/components/timeline/index.js

```diff
@@ -153,6 +161,49 @@ class TimelineSettingsHiddenTracks extends React.PureComponent<{|
   }
 }
 
+class TimelineSettingsActiveTabView extends React.PureComponent<{|
+  +activeBrowsingContextID: BrowsingContextID | null,
+  +showTabOnly: BrowsingContextID | null,
+  +changeShowTabOnly: typeof changeShowTabOnly,
+|}> {
+  _toggleShowTabOnly = () => {
+    const {
+      showTabOnly,
+      changeShowTabOnly,
+      activeBrowsingContextID,
+    } = this.props;
+    if (showTabOnly === null) {
+      changeShowTabOnly(activeBrowsingContextID);
+    } else {
+      changeShowTabOnly(null);
+    }
+  };
+
+  render() {
+    const { activeBrowsingContextID, showTabOnly } = this.props;
+    if (activeBrowsingContextID === null) {
+      return null;
+    }
+
+    return (
+      <form>
+        <div className="timelineSettingsToggle">
+          <label className="photon-label photon-label-micro timelineSettingsToggleLabel">
```

Comment:
Could we size the checkbox to match the checkbox in the _Invert call stack_ in the _Call Tree_ toolbar. It seems larger.